src/main/java/com/inventage/nexusaptplugin/DEBIAN.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Created with IntelliJ IDEA.
  * User: sannies
- * Date: 6/30/12
+ * Date: 20120630
```

Comment:
Actually, I'd rather scrap the whole header oder even better, replace it with an actual comment. The information present now is pretty useless.